app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/ServerSettingsFragment.kt

```diff
@@ -65,5 +67,13 @@ class ServerSettingsFragment : BaseDaggerFragment() {
 
         version.text =
             getString(R.string.version, BuildConfig.VERSION_NAME, BuildConfig.VERSION_CODE)
+
+        viewModel.resetInProgress.observe(viewLifecycleOwner, Observer { resetInProgress ->
+            settingsLogoutBtn.apply {
+                isClickable = !resetInProgress
+                text = if (resetInProgress) "" else resources.getString(R.string.reset_the_app)
```

Comment:
You can replace "" with String(). I think that it looks better.

Reply:
Everyone likes something else ; )